Sources/ValueType.swift

```diff
@@ -49,14 +49,39 @@ extension Int64: ValueType {
 }
 
 extension Array where Element: ValueType {
-    public static func value(from object: Any) throws -> [Element] {
+    public static func value(from object: Any, discardInvalidObjects: Bool = false) throws -> [Element] {
```

Comment:
Yeah, I was thinking about `discardingErrors` too, since I think it helps describe the fact that you're actually throwing away errors that are being thrown.